install/kubernetes/helm/istio-remote/templates/endpoints.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: istio-mixer
+  namespace: istio-system
+subsets:
+- addresses:
+  - ip: {{ .Values.global.mixerIp }}
+  ports:
```

Comment:
Since these ports (and the pilot ports) must match the port values on the cluster that hosts  pilot and mixer these really shouldn't be enumerated separately here,  but derived from the port values used by helm to install the istio control components on the "local" cluster.   Please consider removing these as hard codes and derive or obtain them from 
the other helm charts.

Reply:
i agree - that is followup work.

Reply:
I believe this should be done in this PR itself.  It should be straight forward.